patternfly/patternfly-elements
# fix: pfe-navigation mobile login/language links working

## Get pfe-navigation mobile login/language links working

* Link(s) to demo pages where this element can be viewed:

---

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

* Mobile login and language links should navigate to the URLs specified when clicked or if the user hits enter/spacebar

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. Open the mobile navigation tray
2. Click on the login link
3. Click on the language link
4. Both should navigate away to their URLs
5. Ensure no regressions in trigger items in the main navigation

#### Browser requirements
Your component should work in all of the following environments:


### Ready-for-merge Checklist



**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
